packages/ui/package.json

```diff
@@ -37,6 +37,7 @@
     "preset": "scripts/jest/node"
   },
   "dependencies": {
+    "@floating-ui/react": "^0.24.6",
     "@floating-ui/react-dom-interactions": "^0.13.3",
```

Comment:
please remove this one if not used, if it's used try to replace it with the library you installed

Reply:
done